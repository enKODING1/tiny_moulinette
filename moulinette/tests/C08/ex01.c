#include <stdio.h>
#include "ft_boolean.h"

int main(void)
{
    t_bool i = -2147483648;
    if (i != -2147483648)
        printf("t_bool error\n");
    if (TRUE == 1)
        printf("TRUE is 1.\n");
    if (FALSE == 0)
        printf("FALSE is 0.\n");
    if (SUCCESS == 0)
        printf("SUCCESS is 0.\n");
    if (EVEN(2))
        printf("EVEN(x) is right.\n");
    if (!(EVEN(3)))
        printf("EVEN(x) is right.\n");
    printf("%s", EVEN_MSG);
    printf("%s", ODD_MSG);
    return 0;
}
