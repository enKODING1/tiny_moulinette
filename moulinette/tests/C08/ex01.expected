TRUE is 1.
FALSE is 0.
SUCCESS is 0.
EVEN(x) is right.
EVEN(x) is right.
I have an even number of arguments.
I have an odd number of arguments.
